src/main.cpp

```diff
@@ -362,6 +374,14 @@ int main(int argc, char** argv)
             mkdir((string(getenv("HOME")) + "/.emptyepsilon").c_str(), S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
 #endif
             PreferencesManager::save(string(getenv("HOME")) + "/.emptyepsilon/options.ini");
+        }else if (getenv("APPDATA"))
+        {
+#ifdef _WIN32
+            mkdir((string(getenv("APPDATA")) + "/emptyepsilon").c_str());
+#else
+            mkdir((string(getenv("APPDATA")) + "/emptyepsilon").c_str(), S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
```

Comment:
This feels like happy-go-lucky copypasta: which platform is this supposed to handle?

I don't think APPDATA really is a thing to worry about on anything but windows :)

Reply:
Yeah, copypasta is what it was.  A more eloquent solution would be ideal.